Cython/Debugger/Cygdb.py

```diff
@@ -50,17 +50,21 @@ def make_command_file(path_to_debug_info, prefix_code='',
             try:
                 # Activate virtualenv, if we were launched from one
                 import os
+                import sys
                 virtualenv = os.getenv('VIRTUAL_ENV')
                 if virtualenv:
-                    path_to_activate_this_py = os.path.join(virtualenv, 'bin', 'activate_this.py')
+                    if sys.platform == "win32":
+                        path_to_activate_this_py = os.path.join(virtualenv, 'Scripts', 'activate_this.py')
+                    else:
+                        path_to_activate_this_py = os.path.join(virtualenv, 'bin', 'activate_this.py')
```

Comment:
In conditionals, I often prefer to make it clear that we are always setting the variable, just with different values. That's more difficult to see if you have entirely separate if-branches.
```suggestion
                    scripts_dir = 'Scripts' if sys.platform == "win32" else 'bin'
                    path_to_activate_this_py = os.path.join(virtualenv, scripts_dir, 'activate_this.py')
```